content/01.abstract.md

```diff
@@ -2,6 +2,13 @@
 
 Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
 However, traditional research and multi-author writing processes break down at scale.
-New tools and workflows that rely on automation can catch errors and ensure that contributions are fairly tracked in massively collaborative research.
```

Comment:
The idea that `automation can catch errors` is important and isn't as explicit in the new abstract.

Reply:
I think this can go between the sentence beginning `Manubot adopts...` and `With Manubot...`.

Reply:
Do we think this sentence is now done justice by way of:

> A cloud service automatically evaluates proposed changes to catch errors.

along with 

> precisely track changes over time

and 

> making it transparent

Reply:
Yes